### PR TITLE
RFQ thank you page changes

### LINKED ISSUE #274: RFQ thank you page
Ref

https://www.moleculardevices.com/quote-request-success

image

### PR DESCRIPTION
**Pull request by kapildhiman**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #274 

Test URLs:
- Before: https://main--moleculardevices--hlxsites.hlx.page/quote-request-success
- After: https://rfq-thankyou--moleculardevices--hlxsites.hlx.page/quote-request-success

### CODE REVIEW

**Comment by lydiapuric** [comment]
The font size does not match the mobile original look (too big), otherwise it looks good to me

**Comment by lydiapuric** [comment]
The LHS seems to be effected by LCP. Could you check this please too? See https://pagespeed.web.dev/analysis/https-rfq-thankyou--moleculardevices--hlxsites-hlx-page-quote-request-success/txgapwjdzq?form_factor=mobile
Looks like the background image causes it.

**Review by lydiapuric — APPROVED** [review]
LGTM +1

File: blocks/quote-request/quote-request.css
Comment (mhaack) [inline blocks/quote-request/quote-request.css:1]: Can you please format the file

**Pull request merged**